FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/PasswordPromptView.swift

```diff
@@ -9,26 +10,47 @@ struct PasswordPromptSheet {
 extension PasswordPromptSheet: View {
   var body: some View {
     VStack(spacing: 20) {
-      SecureField(authService.string.passwordInputLabel, text: $password)
-        .textFieldStyle(.roundedBorder)
+      Text(authService.string.confirmPasswordInputLabel)
+        .font(.largeTitle)
+        .fontWeight(.bold)
         .padding()
 
-      HStack {
-        Button(authService.string.cancelButtonLabel) {
-          coordinator.cancel()
-        }
-        Spacer()
-        Button(authService.string.okButtonLabel) {
+      Divider()
+
+      LabeledContent {
+        TextField(authService.string.passwordInputLabel, text: $password)
+          .textInputAutocapitalization(.never)
+          .disableAutocorrection(true)
+          .submitLabel(.next)
+      } label: {
+        Image(systemName: "lock")
+      }.padding(.vertical, 10)
+        .background(Divider(), alignment: .bottom)
+        .padding(.bottom, 4)
+
+      Button(action: {
+        Task {
```

Comment:
Why does this need to be wrapped in a `Task`?

Reply:
Good catch, removed: [`b738cf5` (#1259)](https://github.com/firebase/FirebaseUI-iOS/pull/1259/commits/b738cf53381f82dacb5af64bc429f3ed0066bcd2)